src/ViewHandler/SecuredViewHandler.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace PhpList\RestBundle\ViewHandler;
+
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\View\ViewHandler;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This class is used to add headers to the default response.
+ *
+ * @author Xheni Myrtaj <xheni@phplist.com>
+ */
+class SecuredViewHandler
+{
+    /**
+     * @param ViewHandler $viewHandler
+     * @param View $view
+     * @param Request $request
+     * @param string $format
+     *
+     * @return Response
+     */
+    public function createResponse(ViewHandler $handler, View $view, Request $request, string $format):Response
```

Comment:
Almost there! :-)

Could you please add a space after the `:`?

And now that we have all types as type declarations, the PHPDoc does not offer any additional information, and we can remove it (assuming that PHPCS won't complain).

Reply:
Then let's keep it for this PR. For the mid-term, I'd like to remove all PHPDoc comments that only repeat the PHP (return) type declaration and add nothing, hence reducing the amount of code that needs to be maintained.